Druid-Tests/DruidTestRTLCompiler.class.st

```diff
@@ -28,6 +28,20 @@ cogMethod temporaries do: [ :temp | |args|
 CogObjectRepresentation compile: cogMethod formattedCode classified: 'primitive generators'
 ]
 
+{ #category : #compiling }
+DruidTestRTLCompiler >> compileAbort [
+	"Fake method"
+
+	
+]
+
+{ #category : #compiling }
+DruidTestRTLCompiler >> compileEntry [
+	"Fake method"
+
```

Comment:
After the fixes we did last time I am not sure these two empty methods are required.
Are they?